packages/replayio/src/utils/launch-darkly/initLaunchDarklyFromAccessToken.ts

```diff
@@ -19,6 +22,10 @@ export async function initLaunchDarklyFromAccessToken(accessToken: string) {
       updateLaunchDarklyCache(accessToken, id);
     }
 
+    if (abortSignal.aborted) {
+      return;
+    }
```

Comment:
Primarily that, and also hoping to avoid logging an error from LD init (or any internet requests it makes). Not sure we need this but it seemed safe enough to add.